.github/dependabot.yml

```
@@ -0,0 +1,9 @@
+version: 2
+updates:
+- package-ecosystem: maven
```

```suggestion
- package-ecosystem: gomod
```

`doctl` is a Go project that manages its dependencies using `gomod`.